From 48f9879b148ea87e7a980d0a9b260480f1c2a328 Mon Sep 17 00:00:00 2001
From: "Kevin F. Haggerty" <haggertk@lineageos.org>
Date: Fri, 26 Jan 2018 21:06:19 -0700
Subject: [PATCH 2/5] klte-common: Move ril_{,unsol}_commands_vendor.h to
 include/

* This overlays the skeleton versions in the msm8974-common
  libril.

Change-Id: I0a1be15205de21ed0ced48c685ea69d15b6091eb
---
 include/libril/ril_commands_vendor.h       | 50 ++++++++++++++++++
 include/libril/ril_unsol_commands_vendor.h | 83 ++++++++++++++++++++++++++++++
 libril/ril_commands_vendor.h               | 50 ------------------
 libril/ril_unsol_commands_vendor.h         | 83 ------------------------------
 4 files changed, 133 insertions(+), 133 deletions(-)
 create mode 100644 include/libril/ril_commands_vendor.h
 create mode 100644 include/libril/ril_unsol_commands_vendor.h
 delete mode 100644 libril/ril_commands_vendor.h
 delete mode 100644 libril/ril_unsol_commands_vendor.h

diff --git a/include/libril/ril_commands_vendor.h b/include/libril/ril_commands_vendor.h
new file mode 100644
index 0000000..cd4a1f3
--- /dev/null
+++ b/include/libril/ril_commands_vendor.h
@@ -0,0 +1,50 @@
+/* //device/libs/telephony/ril_commands.h
+**
+** Copyright 2006, The Android Open Source Project
+**
+** Licensed under the Apache License, Version 2.0 (the "License");
+** you may not use this file except in compliance with the License.
+** You may obtain a copy of the License at
+**
+**     http://www.apache.org/licenses/LICENSE-2.0
+**
+** Unless required by applicable law or agreed to in writing, software
+** distributed under the License is distributed on an "AS IS" BASIS,
+** WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+** See the License for the specific language governing permissions and
+** limitations under the License.
+*/
+    {SAMSUNG_REQUEST_BASE, NULL}, // 10000
+    {RIL_REQUEST_DIAL_EMERGENCY_CALL, /* dispatchDial */ NULL}, // 10001
+    {RIL_REQUEST_CALL_DEFLECTION, /* dispatchVoid */ NULL}, // 10002
+    {RIL_REQUEST_MODIFY_CALL_INITIATE, /* dispatchVoid */ NULL}, // 10003
+    {RIL_REQUEST_MODIFY_CALL_CONFIRM, /* dispatchVoid */ NULL}, // 10004
+    {RIL_REQUEST_SET_VOICE_DOMAIN_PREF, /* dispatchVoid */ NULL}, // 10005
+    {RIL_REQUEST_SAFE_MODE, /* dispatchVoid */ NULL}, // 10006
+    {RIL_REQUEST_SET_TRANSMIT_POWER, /* dispatchVoid */ NULL}, // 10007
+    {RIL_REQUEST_GET_CELL_BROADCAST_CONFIG, /* dispatchVoid */ NULL}, // 10008
+    {RIL_REQUEST_GET_PHONEBOOK_STORAGE_INFO, /* dispatchVoid */ NULL}, // 10009
+    {RIL_REQUEST_GET_PHONEBOOK_ENTRY, /* dispatchVoid */ NULL}, // 10010
+    {RIL_REQUEST_ACCESS_PHONEBOOK_ENTRY, /* dispatchVoid */ NULL}, // 10011
+    {RIL_REQUEST_USIM_PB_CAPA, /* dispatchVoid */ NULL}, // 10012
+    {RIL_REQUEST_LOCK_INFO, /* dispatchVoid */ NULL}, // 10013
+    {RIL_REQUEST_STK_SIM_INIT_EVENT, /* dispatchVoid */ NULL}, // 10014
+    {RIL_REQUEST_SET_PREFERRED_NETWORK_LIST, /* dispatchVoid */ NULL}, // 10015
+    {RIL_REQUEST_GET_PREFERRED_NETWORK_LIST, /* dispatchVoid */ NULL}, // 10016
+    {RIL_REQUEST_CHANGE_SIM_PERSO, /* dispatchVoid */ NULL}, // 10017
+    {RIL_REQUEST_ENTER_SIM_PERSO, /* dispatchVoid */ NULL}, // 10018
+    {RIL_REQUEST_SEND_ENCODED_USSD, /* dispatchVoid */ NULL}, // 10019
+    {RIL_REQUEST_CDMA_SEND_SMS_EXPECT_MORE, /* dispatchVoid */ NULL}, // 10020
+    {RIL_REQUEST_HANGUP_VT, /* dispatchVoid */ NULL}, // 10021
+    {RIL_REQUEST_HOLD, /* dispatchVoid */ NULL}, // 10022
+    {RIL_REQUEST_SET_SIM_POWER, /* dispatchVoid */ NULL}, // 10023
+    {10024, NULL}, // 10024
+    {RIL_REQUEST_UICC_GBA_AUTHENTICATE_BOOTSTRAP, /* dispatchVoid */ NULL}, // 10025
+    {RIL_REQUEST_UICC_GBA_AUTHENTICATE_NAF, /* dispatchVoid */ NULL}, // 10026
+    {RIL_REQUEST_GET_INCOMING_COMMUNICATION_BARRING, /* dispatchVoid */ NULL}, // 10027
+    {RIL_REQUEST_SET_INCOMING_COMMUNICATION_BARRING, /* dispatchVoid */ NULL}, // 10028
+    {RIL_REQUEST_QUERY_CNAP, /* dispatchVoid */ NULL}, // 10029
+    {RIL_REQUEST_SET_TRANSFER_CALL, /* dispatchVoid */ NULL}, // 10030
+    {RIL_REQUEST_GET_DISABLE_2G, /* dispatchVoid */ NULL}, // 10031
+    {RIL_REQUEST_SET_DISABLE_2G, /* dispatchVoid */ NULL}, // 10032
+    {RIL_REQUEST_REFRESH_NITZ_TIME, /* dispatchVoid */ NULL}, // 10033
diff --git a/include/libril/ril_unsol_commands_vendor.h b/include/libril/ril_unsol_commands_vendor.h
new file mode 100644
index 0000000..5a026f6
--- /dev/null
+++ b/include/libril/ril_unsol_commands_vendor.h
@@ -0,0 +1,83 @@
+/* //device/libs/telephony/ril_unsol_commands.h
+**
+** Copyright 2006, The Android Open Source Project
+**
+** Licensed under the Apache License, Version 2.0 (the "License");
+** you may not use this file except in compliance with the License.
+** You may obtain a copy of the License at
+**
+**     http://www.apache.org/licenses/LICENSE-2.0
+**
+** Unless required by applicable law or agreed to in writing, software
+** distributed under the License is distributed on an "AS IS" BASIS,
+** WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+** See the License for the specific language governing permissions and
+** limitations under the License.
+*/
+    {SAMSUNG_UNSOL_BASE, /* responseVoid */ NULL, WAKE_PARTIAL}, // 11000, actually UNSOL_RESPONSE_NEW_CB_MSG
+    {RIL_UNSOL_RELEASE_COMPLETE_MESSAGE, /* responseVoid */ NULL, WAKE_PARTIAL}, // 11001
+    {RIL_UNSOL_STK_SEND_SMS_RESULT, /* responseInts */ NULL, WAKE_PARTIAL}, // 11002
+    {RIL_UNSOL_STK_CALL_CONTROL_RESULT, /* responseString */ NULL, WAKE_PARTIAL}, // 11003
+    {11004, NULL, WAKE_PARTIAL}, // 11004
+    {11005, NULL, WAKE_PARTIAL}, // 11005
+    {11006, NULL, WAKE_PARTIAL}, // 11006
+    {11007, NULL, WAKE_PARTIAL}, // 11007
+    {RIL_UNSOL_DEVICE_READY_NOTI, /* responseVoid */ NULL, WAKE_PARTIAL}, // 11008
+    {RIL_UNSOL_GPS_NOTI, /* responseVoid */ NULL, WAKE_PARTIAL}, // 11009
+    {RIL_UNSOL_AM, /* responseString */ NULL, WAKE_PARTIAL}, // 11010
+    {11011, NULL, WAKE_PARTIAL}, // 11011
+    {11012, NULL, WAKE_PARTIAL}, // 11012
+    {RIL_UNSOL_SAP, /* responseRaw */ NULL, WAKE_PARTIAL}, // 11013
+    {11014, NULL, WAKE_PARTIAL}, // 11014
+    {11015, NULL, WAKE_PARTIAL}, // 11015
+    {11016, NULL, WAKE_PARTIAL}, // 11016
+    {11017, NULL, WAKE_PARTIAL}, // 11017
+    {11018, NULL, WAKE_PARTIAL}, // 11018
+    {11019, NULL, WAKE_PARTIAL}, // 11019
+    {RIL_UNSOL_UART, /* responseRaw */ NULL, WAKE_PARTIAL}, // 11020
+    {RIL_UNSOL_SIM_PB_READY, /* responseVoid */ NULL, WAKE_PARTIAL}, // 11021
+    {11022, NULL, WAKE_PARTIAL}, // 11022
+    {11023, NULL, WAKE_PARTIAL}, // 11023
+    {RIL_UNSOL_VE, /* responseRaw */ NULL, WAKE_PARTIAL}, // 11024
+    {11025, NULL, WAKE_PARTIAL},
+    {RIL_UNSOL_FACTORY_AM, /* responseVoid */ NULL, WAKE_PARTIAL}, // 11026
+    {RIL_UNSOL_IMS_REGISTRATION_STATE_CHANGED, /* responseInts */ NULL, WAKE_PARTIAL}, // 11027
+    {RIL_UNSOL_MODIFY_CALL, /* responseVoid */ NULL, WAKE_PARTIAL}, // 11028
+    {11029, NULL, WAKE_PARTIAL}, // 11029
+    {RIL_UNSOL_CS_FALLBACK, /* responseInts */ NULL, WAKE_PARTIAL}, // 11030
+    {11031, NULL, WAKE_PARTIAL}, // 11031
+    {RIL_UNSOL_VOICE_SYSTEM_ID, /* responseInts */ NULL, WAKE_PARTIAL}, // 11032
+    {11033, NULL, WAKE_PARTIAL}, // 11033
+    {RIL_UNSOL_IMS_RETRYOVER, /* responseVoid */ NULL, WAKE_PARTIAL}, // 11034
+    {RIL_UNSOL_PB_INIT_COMPLETE, /* responseVoid */ NULL, WAKE_PARTIAL}, // 11035
+    {11036, NULL, WAKE_PARTIAL}, // 11036
+    {RIL_UNSOL_HYSTERESIS_DCN, /* responseVoid */ NULL, WAKE_PARTIAL}, // 11037
+    {RIL_UNSOL_CP_POSITION, /* responseVoid */ NULL, WAKE_PARTIAL}, // 11038
+    {11039, NULL, WAKE_PARTIAL}, // 11039
+    {11040, NULL, WAKE_PARTIAL}, // 11040
+    {11041, NULL, WAKE_PARTIAL}, // 11041
+    {11042, NULL, WAKE_PARTIAL}, // 11042
+    {RIL_UNSOL_HOME_NETWORK_NOTI, /* responseVoid */ NULL, WAKE_PARTIAL}, // 11043
+    {11044, NULL, WAKE_PARTIAL}, // 11044
+    {11045, NULL, WAKE_PARTIAL}, // 11045
+    {11046, NULL, WAKE_PARTIAL}, // 11046
+    {11047, NULL, WAKE_PARTIAL}, // 11047
+    {11048, NULL, WAKE_PARTIAL}, // 11048
+    {11049, NULL, WAKE_PARTIAL}, // 11049
+    {11050, NULL, WAKE_PARTIAL}, // 11050
+    {11051, NULL, WAKE_PARTIAL}, // 11051
+    {11052, NULL, WAKE_PARTIAL}, // 11052
+    {11053, NULL, WAKE_PARTIAL}, // 11053
+    {RIL_UNSOL_STK_CALL_STATUS, /* responseInts */ NULL, WAKE_PARTIAL}, // 11054
+    {11055, NULL, WAKE_PARTIAL}, // 11055
+    {RIL_UNSOL_MODEM_CAP, /* responseRaw */ NULL, WAKE_PARTIAL}, // 11056
+    {RIL_UNSOL_SIM_SWAP_STATE_CHANGED, /* responseInts */ NULL, WAKE_PARTIAL}, // 11057
+    {RUL_UNSOL_SIM_COUNT_MISMATCHED, /* responseInts */ NULL, WAKE_PARTIAL}, // 11058
+    {11059, NULL, WAKE_PARTIAL}, // 11059
+    {RIL_UNSOL_DUN, /* responseStrings */ NULL, WAKE_PARTIAL}, // 11060
+    {RIL_UNSOL_IMS_PREFERENCE_CHANGED, /* responseInts */ NULL, WAKE_PARTIAL}, // 11061
+    {RIL_UNSOL_SIM_APPLICATION_REFRESH, /* responseInts */ NULL, WAKE_PARTIAL}, // 11062
+    {RIL_UNSOL_UICC_APPLICATION_STATUS, /* responseVoid */ NULL, WAKE_PARTIAL}, // 11063
+    {RIL_UNSOL_VOICE_RADIO_BEARER_HO_STATUS, /* responseInts */ NULL, WAKE_PARTIAL}, // 11064
+    {RIL_UNSOL_CLM_NOTI, /* responseVoid */ NULL, WAKE_PARTIAL}, // 11065
+    {RIL_UNSOL_SIM_ICCID_NOTI, /* responseString */ NULL, WAKE_PARTIAL}, // 11066
diff --git a/libril/ril_commands_vendor.h b/libril/ril_commands_vendor.h
deleted file mode 100644
index cd4a1f3..0000000
--- a/libril/ril_commands_vendor.h
+++ /dev/null
@@ -1,50 +0,0 @@
-/* //device/libs/telephony/ril_commands.h
-**
-** Copyright 2006, The Android Open Source Project
-**
-** Licensed under the Apache License, Version 2.0 (the "License");
-** you may not use this file except in compliance with the License.
-** You may obtain a copy of the License at
-**
-**     http://www.apache.org/licenses/LICENSE-2.0
-**
-** Unless required by applicable law or agreed to in writing, software
-** distributed under the License is distributed on an "AS IS" BASIS,
-** WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-** See the License for the specific language governing permissions and
-** limitations under the License.
-*/
-    {SAMSUNG_REQUEST_BASE, NULL}, // 10000
-    {RIL_REQUEST_DIAL_EMERGENCY_CALL, /* dispatchDial */ NULL}, // 10001
-    {RIL_REQUEST_CALL_DEFLECTION, /* dispatchVoid */ NULL}, // 10002
-    {RIL_REQUEST_MODIFY_CALL_INITIATE, /* dispatchVoid */ NULL}, // 10003
-    {RIL_REQUEST_MODIFY_CALL_CONFIRM, /* dispatchVoid */ NULL}, // 10004
-    {RIL_REQUEST_SET_VOICE_DOMAIN_PREF, /* dispatchVoid */ NULL}, // 10005
-    {RIL_REQUEST_SAFE_MODE, /* dispatchVoid */ NULL}, // 10006
-    {RIL_REQUEST_SET_TRANSMIT_POWER, /* dispatchVoid */ NULL}, // 10007
-    {RIL_REQUEST_GET_CELL_BROADCAST_CONFIG, /* dispatchVoid */ NULL}, // 10008
-    {RIL_REQUEST_GET_PHONEBOOK_STORAGE_INFO, /* dispatchVoid */ NULL}, // 10009
-    {RIL_REQUEST_GET_PHONEBOOK_ENTRY, /* dispatchVoid */ NULL}, // 10010
-    {RIL_REQUEST_ACCESS_PHONEBOOK_ENTRY, /* dispatchVoid */ NULL}, // 10011
-    {RIL_REQUEST_USIM_PB_CAPA, /* dispatchVoid */ NULL}, // 10012
-    {RIL_REQUEST_LOCK_INFO, /* dispatchVoid */ NULL}, // 10013
-    {RIL_REQUEST_STK_SIM_INIT_EVENT, /* dispatchVoid */ NULL}, // 10014
-    {RIL_REQUEST_SET_PREFERRED_NETWORK_LIST, /* dispatchVoid */ NULL}, // 10015
-    {RIL_REQUEST_GET_PREFERRED_NETWORK_LIST, /* dispatchVoid */ NULL}, // 10016
-    {RIL_REQUEST_CHANGE_SIM_PERSO, /* dispatchVoid */ NULL}, // 10017
-    {RIL_REQUEST_ENTER_SIM_PERSO, /* dispatchVoid */ NULL}, // 10018
-    {RIL_REQUEST_SEND_ENCODED_USSD, /* dispatchVoid */ NULL}, // 10019
-    {RIL_REQUEST_CDMA_SEND_SMS_EXPECT_MORE, /* dispatchVoid */ NULL}, // 10020
-    {RIL_REQUEST_HANGUP_VT, /* dispatchVoid */ NULL}, // 10021
-    {RIL_REQUEST_HOLD, /* dispatchVoid */ NULL}, // 10022
-    {RIL_REQUEST_SET_SIM_POWER, /* dispatchVoid */ NULL}, // 10023
-    {10024, NULL}, // 10024
-    {RIL_REQUEST_UICC_GBA_AUTHENTICATE_BOOTSTRAP, /* dispatchVoid */ NULL}, // 10025
-    {RIL_REQUEST_UICC_GBA_AUTHENTICATE_NAF, /* dispatchVoid */ NULL}, // 10026
-    {RIL_REQUEST_GET_INCOMING_COMMUNICATION_BARRING, /* dispatchVoid */ NULL}, // 10027
-    {RIL_REQUEST_SET_INCOMING_COMMUNICATION_BARRING, /* dispatchVoid */ NULL}, // 10028
-    {RIL_REQUEST_QUERY_CNAP, /* dispatchVoid */ NULL}, // 10029
-    {RIL_REQUEST_SET_TRANSFER_CALL, /* dispatchVoid */ NULL}, // 10030
-    {RIL_REQUEST_GET_DISABLE_2G, /* dispatchVoid */ NULL}, // 10031
-    {RIL_REQUEST_SET_DISABLE_2G, /* dispatchVoid */ NULL}, // 10032
-    {RIL_REQUEST_REFRESH_NITZ_TIME, /* dispatchVoid */ NULL}, // 10033
diff --git a/libril/ril_unsol_commands_vendor.h b/libril/ril_unsol_commands_vendor.h
deleted file mode 100644
index 5a026f6..0000000
--- a/libril/ril_unsol_commands_vendor.h
+++ /dev/null
@@ -1,83 +0,0 @@
-/* //device/libs/telephony/ril_unsol_commands.h
-**
-** Copyright 2006, The Android Open Source Project
-**
-** Licensed under the Apache License, Version 2.0 (the "License");
-** you may not use this file except in compliance with the License.
-** You may obtain a copy of the License at
-**
-**     http://www.apache.org/licenses/LICENSE-2.0
-**
-** Unless required by applicable law or agreed to in writing, software
-** distributed under the License is distributed on an "AS IS" BASIS,
-** WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-** See the License for the specific language governing permissions and
-** limitations under the License.
-*/
-    {SAMSUNG_UNSOL_BASE, /* responseVoid */ NULL, WAKE_PARTIAL}, // 11000, actually UNSOL_RESPONSE_NEW_CB_MSG
-    {RIL_UNSOL_RELEASE_COMPLETE_MESSAGE, /* responseVoid */ NULL, WAKE_PARTIAL}, // 11001
-    {RIL_UNSOL_STK_SEND_SMS_RESULT, /* responseInts */ NULL, WAKE_PARTIAL}, // 11002
-    {RIL_UNSOL_STK_CALL_CONTROL_RESULT, /* responseString */ NULL, WAKE_PARTIAL}, // 11003
-    {11004, NULL, WAKE_PARTIAL}, // 11004
-    {11005, NULL, WAKE_PARTIAL}, // 11005
-    {11006, NULL, WAKE_PARTIAL}, // 11006
-    {11007, NULL, WAKE_PARTIAL}, // 11007
-    {RIL_UNSOL_DEVICE_READY_NOTI, /* responseVoid */ NULL, WAKE_PARTIAL}, // 11008
-    {RIL_UNSOL_GPS_NOTI, /* responseVoid */ NULL, WAKE_PARTIAL}, // 11009
-    {RIL_UNSOL_AM, /* responseString */ NULL, WAKE_PARTIAL}, // 11010
-    {11011, NULL, WAKE_PARTIAL}, // 11011
-    {11012, NULL, WAKE_PARTIAL}, // 11012
-    {RIL_UNSOL_SAP, /* responseRaw */ NULL, WAKE_PARTIAL}, // 11013
-    {11014, NULL, WAKE_PARTIAL}, // 11014
-    {11015, NULL, WAKE_PARTIAL}, // 11015
-    {11016, NULL, WAKE_PARTIAL}, // 11016
-    {11017, NULL, WAKE_PARTIAL}, // 11017
-    {11018, NULL, WAKE_PARTIAL}, // 11018
-    {11019, NULL, WAKE_PARTIAL}, // 11019
-    {RIL_UNSOL_UART, /* responseRaw */ NULL, WAKE_PARTIAL}, // 11020
-    {RIL_UNSOL_SIM_PB_READY, /* responseVoid */ NULL, WAKE_PARTIAL}, // 11021
-    {11022, NULL, WAKE_PARTIAL}, // 11022
-    {11023, NULL, WAKE_PARTIAL}, // 11023
-    {RIL_UNSOL_VE, /* responseRaw */ NULL, WAKE_PARTIAL}, // 11024
-    {11025, NULL, WAKE_PARTIAL},
-    {RIL_UNSOL_FACTORY_AM, /* responseVoid */ NULL, WAKE_PARTIAL}, // 11026
-    {RIL_UNSOL_IMS_REGISTRATION_STATE_CHANGED, /* responseInts */ NULL, WAKE_PARTIAL}, // 11027
-    {RIL_UNSOL_MODIFY_CALL, /* responseVoid */ NULL, WAKE_PARTIAL}, // 11028
-    {11029, NULL, WAKE_PARTIAL}, // 11029
-    {RIL_UNSOL_CS_FALLBACK, /* responseInts */ NULL, WAKE_PARTIAL}, // 11030
-    {11031, NULL, WAKE_PARTIAL}, // 11031
-    {RIL_UNSOL_VOICE_SYSTEM_ID, /* responseInts */ NULL, WAKE_PARTIAL}, // 11032
-    {11033, NULL, WAKE_PARTIAL}, // 11033
-    {RIL_UNSOL_IMS_RETRYOVER, /* responseVoid */ NULL, WAKE_PARTIAL}, // 11034
-    {RIL_UNSOL_PB_INIT_COMPLETE, /* responseVoid */ NULL, WAKE_PARTIAL}, // 11035
-    {11036, NULL, WAKE_PARTIAL}, // 11036
-    {RIL_UNSOL_HYSTERESIS_DCN, /* responseVoid */ NULL, WAKE_PARTIAL}, // 11037
-    {RIL_UNSOL_CP_POSITION, /* responseVoid */ NULL, WAKE_PARTIAL}, // 11038
-    {11039, NULL, WAKE_PARTIAL}, // 11039
-    {11040, NULL, WAKE_PARTIAL}, // 11040
-    {11041, NULL, WAKE_PARTIAL}, // 11041
-    {11042, NULL, WAKE_PARTIAL}, // 11042
-    {RIL_UNSOL_HOME_NETWORK_NOTI, /* responseVoid */ NULL, WAKE_PARTIAL}, // 11043
-    {11044, NULL, WAKE_PARTIAL}, // 11044
-    {11045, NULL, WAKE_PARTIAL}, // 11045
-    {11046, NULL, WAKE_PARTIAL}, // 11046
-    {11047, NULL, WAKE_PARTIAL}, // 11047
-    {11048, NULL, WAKE_PARTIAL}, // 11048
-    {11049, NULL, WAKE_PARTIAL}, // 11049
-    {11050, NULL, WAKE_PARTIAL}, // 11050
-    {11051, NULL, WAKE_PARTIAL}, // 11051
-    {11052, NULL, WAKE_PARTIAL}, // 11052
-    {11053, NULL, WAKE_PARTIAL}, // 11053
-    {RIL_UNSOL_STK_CALL_STATUS, /* responseInts */ NULL, WAKE_PARTIAL}, // 11054
-    {11055, NULL, WAKE_PARTIAL}, // 11055
-    {RIL_UNSOL_MODEM_CAP, /* responseRaw */ NULL, WAKE_PARTIAL}, // 11056
-    {RIL_UNSOL_SIM_SWAP_STATE_CHANGED, /* responseInts */ NULL, WAKE_PARTIAL}, // 11057
-    {RUL_UNSOL_SIM_COUNT_MISMATCHED, /* responseInts */ NULL, WAKE_PARTIAL}, // 11058
-    {11059, NULL, WAKE_PARTIAL}, // 11059
-    {RIL_UNSOL_DUN, /* responseStrings */ NULL, WAKE_PARTIAL}, // 11060
-    {RIL_UNSOL_IMS_PREFERENCE_CHANGED, /* responseInts */ NULL, WAKE_PARTIAL}, // 11061
-    {RIL_UNSOL_SIM_APPLICATION_REFRESH, /* responseInts */ NULL, WAKE_PARTIAL}, // 11062
-    {RIL_UNSOL_UICC_APPLICATION_STATUS, /* responseVoid */ NULL, WAKE_PARTIAL}, // 11063
-    {RIL_UNSOL_VOICE_RADIO_BEARER_HO_STATUS, /* responseInts */ NULL, WAKE_PARTIAL}, // 11064
-    {RIL_UNSOL_CLM_NOTI, /* responseVoid */ NULL, WAKE_PARTIAL}, // 11065
-    {RIL_UNSOL_SIM_ICCID_NOTI, /* responseString */ NULL, WAKE_PARTIAL}, // 11066
-- 
2.7.4

